code/game/turfs/portal.dm

```diff
@@ -33,7 +33,10 @@
 /turf/portal/update_icon()
 	overlays.Cut()
 	vis_contents.Cut()
-	target_turf = locate(src.x+teleport_x,src.y+teleport_y,src.z+teleport_z)
+	var/turf/temp_turf = locate(src.x+teleport_x,src.y+teleport_y,src.z+teleport_z)
+	if(istype(temp_turf,/turf/portal))
+		return
```

Comment:
It should probably emit a warning rather than failing silently

Reply:
Warning then fail or just warning?

Reply:
Warning then fail. Still don't want this to happen, it would just be better if it notified someone of the mistake.

Reply:
Okay, done